docs/docs/creating-global-styles.md

```diff
@@ -2,7 +2,130 @@
 title: Creating global styles
 ---
 
-In this guide, we'll walk through using traditional global styles in your site.
+In nearly every site, there will be some global styles, such as a reset or typography defaults. This guide will walk through how to add global styles to your site, whether you use standard `.css` files (or with preprocessors like Sass/LESS) or a CSS-in-JS solution.
+
+## Table of Contents
+
+- [How to add global styles in Gatsby with standard CSS files](#how-to-add-global-styles-in-gatsby-with-standard-css-files)
+- [How to add global styles in Gatsby using CSS-in-JS](#how-to-add-global-styles-in-gatsby-using-css-in-js)
+- [Add global styles with CSS files and no layout component](#add-global-styles-with-css-files-and-no-layout-component)
+
+## How to add global styles in Gatsby with standard CSS files
+
+The best way to add global styles is with a [shared layout component](https://next.gatsbyjs.org/tutorial/part-three/#your-first-layout-component). This layout component is used for things that are shared throughout the site, including styles, header components, and other common items.
+
+> _NOTE:_ This pattern is implemented by default in [the default starter](https://github.com/gatsbyjs/gatsby-starter-default).
```

Comment:
Not a big deal, but could link directly to the layout file here instead.

Reply:
Good call!